primefaces/src/main/java/org/primefaces/util/FileUploadUtils.java

```diff
@@ -69,6 +69,12 @@ public class FileUploadUtils {
 
     private static final Logger LOGGER = Logger.getLogger(FileUploadUtils.class.getName());
 
+    // Message keys
+    private static final String FILENAME_EMPTY = "primefaces.FileValidator.FILENAME_NOT_NULL";
```

Comment:
also change the string please

Reply:
Fixed also updated the localization.md docs